repondeur/zam_repondeur/fetch/__init__.py

```diff
@@ -71,6 +71,10 @@ def get_article_num_mult(title: str) -> Tuple[str, str]:
     title = title.replace("1er", "1").replace("liminaire", "0")
     if " " in title:
         num, mult = title.split(" ", 1)
+        try:
+            int(num)
+        except ValueError:
+            return "", ""
```

Comment:
This is a problem in case there are many annexes given the following `get_one_or_create` will be unable to make the distinction (same empty num/mult), any idea?